src/components/ProjectStatus/ProjectStatus.js

```diff
@@ -0,0 +1,19 @@
+import "./ProjectStatus.scss";
```

Comment:
Wondering whether `ProjectStatus` is the most intuitive name for this? Maybe something like `MobileProjectBar` would reflect the fact that it's the replacement of the `ProjectBar` on mobile?

Reply:
re-named as well